spec.bs

```diff
@@ -810,6 +810,83 @@ To <dfn>compute protocol matches a special scheme flag</dfn> given a [=construct
   1. If the result of running [=protocol component matches a special scheme=] given |protocol component| is true, then set |parser|'s [=constructor string parser/protocol matches a special scheme flag=] to true.
 </div>
 
+<h2 id=urlpatternlist-class>The {{URLPatternList}} class </h2>
+
+<xmp class="idl">
+[Exposed=(Window,Worker)]
+interface URLPatternList {
+  constructor(sequence<URLPatternListEntry> entries);
```

Comment:
I think some of the feedback in https://github.com/WICG/urlpattern/issues/30 suggested it would be nice to be able to dynamically mutate the list.  So add new entries to the list and possibly delete entries from the list.

Reply:
Makes sense. I didn't want to add this right away until we figure out the internal sort order a little better.